tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -509,9 +534,162 @@ internal static bool IsAlpine()
                 envVars["COR_PROFILER_PATH_ARM64"] = tracerProfilerArm64;
             }
 
+            if (!string.IsNullOrEmpty(devPath))
+            {
+                envVars["DEVPATH"] = devPath;
+            }
+
             return envVars;
         }
 
+        private static bool EnsureDatadogTraceIsInTheGac(string tracerHome)
+        {
+            // We try to ensure Datadog.Trace.dll is installed in the gac for compatibility with .NET Framework fusion class loader
+            // Let's find gacutil, because CI Visibility runs with the SDK / CI environments it's probable that's available.
+            Log.Debug("EnsureDatadogTraceIsInTheGac: Looking for gacutil");
+            var cmdResponse = ProcessHelpers.RunCommand(new ProcessHelpers.Command("where", "gacutil"));
```

Comment:
I don't know how hard you want to try, but for reference the VS developer prompt looks for `\Microsoft\Microsoft SDKs\Windows` in `HKLM\SOFTWARE`, `HKCU\SOFTWARE`, and `HKCU\SOFTWARE\Wow6432Node` to list the installed SDKs (and check their `InstallationFolder` key for the path)

Reply:
Yeah, I want to avoid accessing the register for now, at least for this first approach. We can always add it later in case we really need it.